CHANGELOG.md

```diff
@@ -967,9 +967,14 @@
 - Ensure that activities validate the presence of `planned_start_date` (or `actual_start_date`) and `planned_end_date` (or `actual_end_date`)
 - Add data migrations for fixing inconsistencies with implementing organisations: add the implementing role to organisations that are missing it; normalise organisation names to uppercase; remove excess spaces from organisation names
 
+## [release-97] 2022-02-15
+
+- Upgrade framework dependencies (gems)
```

Comment:
https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-96...release-97 also has a [Drop the legacy 'implementing_organisations' table, no longer used](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/commit/66ce6ce9dfd8c18e6b0bd2cc6ef3fc7fad98e338) commit, which is worth documenting in my opinion

Reply:
We agreed via Slack that it wasn't a user-facing change, therefore not relevant for the usual purpose of the changelog.